packages/client/modules/directives.ts

```diff
@@ -45,7 +45,7 @@ export default function createDirectives() {
             elements.value.push(el)
 
           // Set default dir.value
-          if (dir.value === null)
+          if (dir.value === undefined)
```

Comment:
```suggestion
          if (dir.value == null)
```
Maybe this to capture both null and undefined?